Unit1_Lesson2.3_Aggregates.sql

```diff
@@ -0,0 +1,62 @@
+-- Drill: Using SQL to pull aggregated data from the Bay Area Bikeshare dataset 
+
+-- 1. What was the hottest day in our data set? Where was that?
+-- 2. How many trips started at each station?
+-- 3. What's the shortest trip that happened? 
+-- 4. What is the average trip duration, by end station?
+
+-- 1. The day with the highest mean temperature (94F) was September 11th, 2015, at zipcode 94063. The "hottest day" could also be interpreted as the day with the highest maximum temperature, in which case the hottest day was November 17th, 2015 at 94063, with a maximum temperature of 134F. (This value is suspect, since a temperature of 134F during autumn in the Bay Area is unlikely!)
+
+SELECT
+    date,
+	ZIP,
+	MAX(MeanTemperatureF) MeanTemperatureF
+FROM
+    weather
+GROUP by date
+ORDER BY MeanTemperatureF DESC
```

Comment:
When you are using ORDER BY and LIMIT 1, the MAX command is redundant (and vice versa)